options/ansi/include/mlibc/file-io.hpp

```diff
@@ -76,7 +76,7 @@ struct abstract_file : __mlibc_file_base {
 };
 
 struct fd_file : abstract_file {
-	fd_file(int fd, void (*do_dispose)(abstract_file *) = nullptr);
+	fd_file(int fd, void (*do_dispose)(abstract_file *) = nullptr, bool is_unbuffered = false);
```

Comment:
Rename `is_unbuffered` -> `force_unbuffered`.

Reply:
I’ll perform the rename when I’m back home again.